src/_typography.scss

```diff
@@ -1 +1,41 @@
+// typography.scss
 
+// i have defined my variables for the typography, here we have adjustments to each heading tag and the body text, and also provided a new font family default. Also adding font weight and heigh to the heading tags to make them pop out more.
+$font-family-default: "Times New Roman", Times, serif;
```

Comment:
Could you please update the font to **Helvetica Neue** instead of Times for consistency with the overall design?

Reply:
That's fine. Just change the Times to Helvetica Neue . 
Thanks,
Zoey